lib/pbio/include/pbdrv/bluetooth.h

```diff
@@ -151,6 +153,7 @@ typedef struct {
     uint8_t status;
     uint8_t bdaddr[6];
     char name[20];
+    lwp3_hub_kind_t hub_kind;
```

Comment:
Moving this to be the first item in the struct should make the compiled code size a bit smaller.